### DISCUSSION TITLE
Allow seamless link to non-Nova application from Nova without opening a new tab

### DISCUSSION THREAD

**Discussion by paullexen** [laravel/nova-issues#4510] [Ideas & Feature Requests]
I want to include a menu item that links directly back to my "main" application from within Nova. My goal is for my users to move seamlessly between the regular app and the admin area. Linking to Nova is no problem, but linking directly back to the application (without opening a new tab) seems overly difficult...or maybe not possible (unless I'm missing something).

Here's my menu:

![image](https://user-images.githubusercontent.com/4392267/176714345-01afcb2c-6345-4276-850f-30a590596390.png)

Here's how I'm creating the menu:

```
Nova::mainMenu(function (Request $request) {
    return [
        MenuSection::make('Resources', [
            MenuItem::resource(User::class),
            MenuItem::resource(Project::class),
        ]),
        MenuItem::externalLink('Back to Application', '/')
    ];
});
```

Creating an external link opens the main application in a new tab. Is there any way to create a link without opening a new tab? If not, it would be nice to specify the target. Something like: 

```
MenuItem::externalLink('Back to Application', '/')->openNew(false);
```
or
```
MenuItem::externalLink('Back to Application', '/')->setTarget('');
```

**Comment by stf-alexander** [comment]
Having the same problem right now!
